experimental/uefi/src/main.rs

```diff
@@ -17,13 +17,24 @@
 #![no_main]
 #![no_std]
 #![feature(abi_efiapi)]
+#![feature(never_type)]
 #![feature(custom_test_frameworks)]
 // As we're in a `no_std` environment, testing requires special handling. This
 // approach was inspired by https://os.phil-opp.com/testing/.
 #![test_runner(crate::test_runner)]
 #![reexport_test_harness_main = "test_main"]
 
-use uefi::{prelude::*, table::runtime::ResetType, ResultExt};
+#[macro_use]
+extern crate log;
```

Comment:
Just a note for future consideration: Using the log crate is fine for now as we are still early in the development. The actual runtime will disable the log crate to ensure that other libraries don't accidentally log sensitive information and use a custom logger.